workloads/README.md

```diff
@@ -0,0 +1,56 @@
+## Deploying AgnosticD workloads
+
+This is a collection of scripts that help you deploy workloads on AgnosticD Openshift clusters. It includes sample configuration files for migration related workloads.
+
+### NOTE
+
+All workloads should be available in AgnosticD repository. However, the migration related workloads are yet not part of the upstream repo. Please checkout following repo to deploy migration workloads.
+
+```bash
+git clone https://github.com/pranavgaikwad/agnosticd
+```
+
+### Deploying a workload
+
+To deploy a workload, you can use the `deploy_workload.sh` script.
+
+#### Usage
+
+To create a new workload, run following script 
+
+```bash
+./deploy_workload.sh -a create -w <workload_name> -v <ocp_version>
+```
+
+`workload_name` is the name of the workload to deploy. 
+
+We have two migration workloads -
+
+* migration : Mig Operator workload to deploy UI, Controller and Velero
+* mssql : A sample MsSQL server with a Node.js app 
+
+To remove the workload, run following script
+
+```bash
+./deploy_workload.sh -a remove -w <workload_name> -v <ocp_version> 
+```
+
+For example, to deploy migration workload on Openshift 3 cluster, run
+
+```bash
+./deploy_workload.sh -a create -w migration -v 3
+```
+
+For more information about the script, run 
+
+```bash
+./deploy_workload.sh -h
+```
+
+### About Workload Configuration
+
+By default, all the workloads are launched with their default configurations. For migration scenarios, there is no need to change any configuration. 
```

Comment:
```
By default, all the workloads are launched with their default configurations.
```
I think we can delete this sentence, seems obvious that workloads will launch with "their default configuration"

Reply:
```
For migration scenarios, there is no need to change any configuration. 
```
->>
```
Migration workloads have default variables set in `workload_vars/<workload_name>.yml`. 

You may change these variables for your use-case, but the defaults will allow for for Migration of apps from OpenShift 3->4 with the Migration Controller + UI located on OpenShift 4.